presto-native-execution/presto_cpp/main/operators/LocalPersistentShuffle.h

```diff
@@ -84,14 +84,15 @@ class LocalPersistentShuffleWriter : public ShuffleWriter {
 
   void noMoreData(bool success) override;
 
+  folly::F14FastMap<std::string, int64_t> stats() const override {
```

Comment:
There could be dimensions that we'd like to report in the future, whose type are not integers (e.g.: string). Is it possible to make the interface more flexible on typing? Thanks!

Reply:
This would require changing the operator stats. What are some examples you are thinking about? I would suggest to add support for integer counters first and consider other types of counters later.

Reply:
One challenge with non-numeric stats is how to aggregate these.

Reply:
Some non-integer metrics like this: https://fburl.com/scuba/cosco_native_client_reader_stats/rszer4c5